test/orbitdb-replication.test.js

```diff
@@ -6,7 +6,7 @@ import config from './config.js'
 import connectPeers from './utils/connect-nodes.js'
 import waitFor from './utils/wait-for.js'
 
-describe('Replicating databases', function () {
+describe('OrbitDB - Replicating databases', function () {
```

Comment:
We can prolly leave these "OrbitDB" prefixes out given we're running the test implicitly in "orbitdb" project.